### PR TITLE
Add CASE

### PR DESCRIPTION
**Pull request by SebConejo**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by nodiscc** [comment]
Hi,
We only list end user applications, not frameowrks/libraries/SDKs.

**Comment by SebConejo** [comment]
Hi @nodiscc .

Thank you for your quick reply. 

I understood that. But I think there is a misunderstood regarding CASE. This is a backend-as-a-Service like appwrite or Pocketbase. If these tools are legitimately listed, CASE should be too.

**Comment by nodiscc** [comment]
Thanks for the heads-up @SebConejo, I will take another look at Appwrite and Pocketbase to determine if they should actually be listed.

File: software/case.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: CASE
+website_url: https://case.app
+source_code_url: https://github.com/casejs/case
+description: Instant backend in TS with DB, Admin panel, API, JS SDK
+licenses:
+  - MIT
+platforms:
```
Comment (ryangurn) [inline software/case.yml:7]: These do not appear to be valid platforms.

File: software/case.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  - Typescript
+  - Nodejs
+tags:
+  - Software Development - Low Code - Backend
```
Comment (ryangurn) [inline software/case.yml:11]: The formatting here does not look correct.

**Review by ryangurn — CHANGES_REQUESTED** [review]
Can you please fix the platforms and ensure that the tags are correctly formatted?

**Comment by nodiscc** [comment]
@ryangurn Don't mind the errors, I'm just keeping this PR open to remind me to 

> take another look at Appwrite and Pocketbase to determine if they should actually be listed.

(see comments). But I should probably open an issue instead.

File: software/case.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: CASE
+website_url: https://case.app
+source_code_url: https://github.com/casejs/case
+description: Instant backend in TS with DB, Admin panel, API, JS SDK
+licenses:
+  - MIT
+platforms:
+  - Javascript
```
Comment (nodiscc) [inline software/case.yml:8]:
```suggestion
  - Nodejs
```

File: software/case.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+name: CASE
+website_url: https://case.app
+source_code_url: https://github.com/casejs/case
+description: Instant backend in TS with DB, Admin panel, API, JS SDK
```
Comment (nodiscc) [inline software/case.yml:4]:
```suggestion
description: Lightweight BaaS (Backend As A Service) requiring minimal coding, with database, admin panel, API, and Javascript SDK.
```

**Review by nodiscc — APPROVED** [review]
Since this procudes a basic working application without writing any code, I've changed my mind and decided it could be included. The unwritten guideline about SDKs/libraries not being in scope for the list needs to be refined and written down.

```bash
nvm use 18
npx create-case-app my-case-app
-> working backend/admin panel at http://localhost:4000/auth/login
```

**Pull request merged**